app/sass/components/_pages-search.scss

```diff
@@ -140,21 +206,18 @@ $ancho:$unidad * 2;
   content:"\e615";
       content: "\e626";
 }
-.browse-filters-block > ul {
-  max-height: 200px;
-  overflow: auto;
-  label, input {
```

Comment:
@mmilaprat @bullz what do you think about the cursor style for the facet check-box and label?